src/grt/src/GlobalRouter.tcl

```diff
@@ -314,7 +315,13 @@ proc repair_antennas { args } {
       sta::check_positive_integer "-iterations" $iterations
     }
 
-    grt::repair_antennas $diode_mterm $iterations
+    set par_margin 0
+    if { [info exists keys(-par_margin)] } {
+      set par_margin $keys(-par_margin)
+      sta::check_positive_float "-par_margin" $par_margin
```

Comment:
I am guessing the parameter is a percentage, which for consistency with the resizer margins should be between 0 and 100. There is no check on the range here. See src/rsz/src/Resizer.tcl repair_design.
Also the README is not updated to doc the arg.